network/basics/uri.py

```diff
@@ -153,20 +153,24 @@
 '''
 
 EXAMPLES = '''
-# Check that you can connect (GET) to a page and it returns a status 200
-- uri: url=http://www.example.com
+- name: Check that you can connect (GET) to a page and it returns a status 200
+  uri:
+    url: 'http://www.example.com'
 
 # Check that a page returns a status 200 and fail if the word AWESOME is not
 # in the page contents.
-- action: uri url=http://www.example.com return_content=yes
+- uri:
+    url: http://www.example.com
```

Comment:
Not sure of the _exacet_ rule around this, but my testing shows it works with or without quotes and doesn't report a syntax error.